File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/abstractClasses/Food.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses;

import it.paranoidsquirrels.idleguildmaster.R;

/* JADX INFO: loaded from: classes3.dex */
public abstract class Food extends Consumable {
    protected transient int feedPower;

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable, it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    public int printType() {
        return R.string.type_food;
    }

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable
    public int printConsumeImage() {
        return R.drawable.consume_food;
    }

    public int getFeedPower() {
        return this.feedPower;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/abstractClasses/Food.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses

import it.paranoidsquirrels.idleguildmaster.R

abstract class Food : Consumable() {
    @JvmField @Transient protected var feedPower: Int = 0

    override fun printType(): Int = R.string.type_food

    override fun printConsumeImage(): Int = R.drawable.consume_food

    open fun getFeedPower(): Int = feedPower
}
